concent_api/core/models.py

```diff
@@ -698,7 +698,29 @@ class PendingEthereumTransaction(Model):
     created_at = DateTimeField(auto_now_add=True)
 
 
+class DepositAccountManager(Manager):
+
+    def get_or_create_full_clean(self, client: bytes, ethereum_address: str) -> 'DepositAccount':
```

Comment:
Maybe it would be better to pass all `args` and `kwargs` to the `DepositAccount` constructor instead of listing them explicitly here?

Reply:
Not in this case, because I then want to query by those two specific fields, and I need to check if they are set. In other words, in this case it is better to point out that exactly those two are necessary.

Reply:
But then it would be possible to use `created_at` there too, and in case when object does not exist, this `created_at` would be passed to new object.

Reply:
Yeah, but it's the same when you call normal `create()`. If you give it `created_at`, it will save it.

Reply:
Anyway, maybe what I propose does have other disadvantages. Never mind. What you have here is a valid solution too.